hybridse/src/udf/default_udf_library.cc

```diff
@@ -666,9 +666,39 @@ void DefaultUdfLibrary::InitStringUdf() {
                 select date_format(date(1590115420000),"%Y-%m-%d");
                 --output "2020-05-22"
             @endcode)");
+    RegisterExternal("upper")
+        .args<codec::StringRef>(reinterpret_cast<void*>(
+            static_cast<void (*)(codec::StringRef*, codec::StringRef*, bool*)>(udf::v1::upper)))
+        .return_by_arg(true)
+        .returns<Nullable<codec::StringRef>>()
+        .doc(R"(
+            @brief Convert all the characters to uppercase.
```

Comment:
doc the default locale used in case confusion. though I don't found the reference yet

Reply:
We might need to refactor .doc() interface.
Now the function documenting is a little bit confusing, especially when the developer alias a registered function.
We should find a way to document the alias function.

Reply:
fwd #807